### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by sv-dubov**

### CODE REVIEW

File: src/makeRobotAccountant.js [hunk 1]
```diff
@@ -6,7 +6,19 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let count = 0;
+
+  return function(firstNumber) {
```
Comment (andrieieva-yuliia) [inline src/makeRobotAccountant.js]: you can write without 'function' 
I mean `return (firstNumber)`

File: src/makeRobotAccountant.js [hunk 2]
```diff
@@ -6,7 +6,19 @@
...
+    return function(secondNumber) {
+      count++;
+
+      if (count > 3 && count % 2 === 0) {
```
Comment (andrieieva-yuliia) [inline src/makeRobotAccountant.js]: you can use ternary operator here

**Review by andrieieva-yuliia — APPROVED** [review]

File: src/makeRobotAccountant.js (hunk 1 above)
Comment (sv-dubov) [reply src/makeRobotAccountant.js]: Thanks, fixed

File: src/makeRobotAccountant.js (hunk 2 above)
Comment (sv-dubov) [reply src/makeRobotAccountant.js]: Yes, thsnks, but I'm not really sure that its needed here. Will wait for mentor's decision :)

Comment (denys-cheporniuk) [reply src/makeRobotAccountant.js]: I agree with @andrieieva-yuliia, ternary operator is better here

**Review by denys-cheporniuk — CHANGES_REQUESTED** [review]

**Review by denys-cheporniuk — APPROVED** [review]

**Pull request closed without merging**